Frends.PDF.Create/Frends.PDF.Create/Frends.PDF.Create.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <Version>1.0.0</Version>
+    <Authors>Frends</Authors>
```

Comment:
Author name could be you? Or?

Reply:
We haven't discussed about this, but at least in the guidelines the author is Frends. I don't think it is necessary to mark individual people as authors.